src/Chapter1.hs

```diff
@@ -551,7 +555,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c 
+  | c == 'A' = True
+  | c == 'E' = True
+  | c == 'I' = True
+  | c == 'O' = True
+  | c == 'U' = True
```

Comment:
Oh, that's actually a good point that uppercased vowels are vowels as well. But for this task, we thought about checking only lowercase ones 😄